about.html

```diff
@@ -0,0 +1,132 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>About Page</title>
+</head>
+<body>
+    <header>
+        <div id="blackHeader">
+            <div class="blackIconsContainer">
+                <a href=""><img class="blackHeaderIcon" src="./images/facebookIcon.png" alt=""></a>
+                <a href=""><img class="blackHeaderIcon" src="./images/twitterIcon.png" alt=""></a>
+            </div>
+            <a href=""><div class="blackHeaderOption">English</div></a>
+            <a href=""><div class="blackHeaderOption">My Page</div></a>
+            <a href=""><div class="blackHeaderOption">Logout</div></a>
+        </div>
+        <div id="whiteHeader">
+            <a class="logoLink" href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./index.html"><div>Home</div></a>
+                <a class="menuOptionsLinks" href="./index.html"><div>Event</div></a>
+                <a class="menuOptionsLinks" href="./index.html#programDetails"><div>Join</div></a>
+                <a class="menuOptionsLinks sponsorMobile" href="#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks sponsorDesk" href="./index.html#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks" href="#"><div>News</div></a>
+                <a class="menuOptionsLinks ccCampaign" href="#"><div>CC Campaign</div></a>
+            </div>
+        </div>
+    </header>
+                                <!-- Main Section -->
+    <section class="mainAbout">
+        <nav class="hamburgerMenu">
+            <span class="bar bar1"></span>
+            <span class="bar bar2"></span>
+            <span class="bar bar3"></span>
+        </nav>
+        <img class="iconsMainAbout burrito" src="./images/iconsMxImages/9burrito-removebg-preview.png" alt="">
+        <img class="iconsMainAbout trumpet" src="./images/iconsMxImages/8trompet-removebg-preview.png" alt="">
+        <div id="mainAboutTextContainer">
+            <div id="helloAndEventContainerAbout">
+                <div id="hello">"Hello! Food Lovers"</div>
+                <h1 id="event">INTERNATIONAL MEXICAN FOOD CONVENTION 2023</h1>
+                <span class="underline underEvent"></span>
+            </div>
+            <span>
+            <img class="iconsMainAbout taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+            <img class="iconsMainAbout maracas" src="./images/iconsMxImages/4mar-removebg-preview.png" alt="">
+            <img class="iconsMainAbout mexa" src="./images/iconsMxImages/10mexicanPerson-removebg-preview.png" alt="">
+            </span>
+            <p id="mainParagraphMobile">The Global Summit brings together the community of  
```

Comment:
- [optional] please, i advise you add your font size a little, so your work can be more user-friendly.. Remember, If it is too difficult to read, your audience may just skip through
![Screenshot (1205)](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/65298048/a4e18a36-3db6-4733-9a12-35da13f405b6)

Reply:
Hi @tufoinnkuo10 you mean a little more size right?

Reply:
@tufoinnkuo10 or what do you mean with "add a little", please let me know